### DISCUSSION TITLE
Upstream Source Code Is Not Free?

### DISCUSSION THREAD

**Discussion by seniorm0ment** [osmandapp/OsmAnd#13863] [General]
F-Droid recently added an anti-feature message to the OSM page stating the upstream source is not free. I do not remember it having this.
could any clarification be made on this? is the upstream no long free? I notice it still states gplv3 so i'm confused.

**Comment by vshcherb** [comment]
OsmAnd cloud is not free because it uses server side storage. Everything else stays the same
